src/main/java/racingwinner/RacingGame.java

```diff
@@ -0,0 +1,88 @@
+package racingwinner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingGame {
+    private int rounding;
+    private List<Car> allCars;
+    private List<Car> winners = new ArrayList<>();
+    private final int BOUND = 10;
+    private final int MOVE_CONDITION = 4;
+
+    public RacingGame(String[] carNames, int rounds) {
+        this.rounding = rounds;
+        this.allCars = new ArrayList<>(carNames.length);
+
+        for (int idx = 0; idx < carNames.length; idx++) {
+            Car racingCar = new Car(carNames[idx]);
+            allCars.add(racingCar);
+        }
+    }
+
+    public void racing() {
+        for (int idx = 0; idx < this.rounding; idx++) {
+            moveCar();
+        }
+        winner();
+    }
+
+    private void moveCar() {
+        for (int carIdx = 0; carIdx < this.allCars.size(); carIdx++) {
+            Car nowCar = allCars.get(carIdx);
+            isGreaterThan4(nowCar);
+        }
+        System.out.println();
+    }
+
+    private int getRandomNum() {
+        Random random = new Random();
+        return random.nextInt(BOUND);
+    }
+
+    public Boolean moveOrNot(int randomNum) {
```

Comment:
참조 자료형 대신 기본 자료형을 사용하면 어떨까요? 아래의 글을 참고하세요.
https://jaehun2841.github.io/2019/03/01/effective-jave-item61

```suggestion
    public boolean moveOrNot(int randomNum) {
```